test/tests/ctypes_test.py

```diff
@@ -0,0 +1,8 @@
+# skip-if: True
```

Comment:
I don't think it will working now. I saw @undingen did some fixs in a certain commit, in that commit, it could pass, but seems it not included in the current #937 . @undingen , would you mind to take a look, please?